apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import styled from '@emotion/styled';
+import { useMantineColorScheme } from '@mantine/core';
+import { colors } from '../../design-system';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useEffectOnce } from '../../hooks';
+import { useProviders } from './useProviders';
+import { IIntegratedProvider } from './IntegrationsStoreModal';
+
+export function UpdateProviderPage() {
+  const [selectedProvider, setSelectedProvider] = useState<IIntegratedProvider | null>(null);
+  const { providers } = useProviders();
+  const { colorScheme } = useMantineColorScheme();
+  const isDark = colorScheme === 'dark';
+  const { integrationId } = useParams();
+  const navigate = useNavigate();
+
+  useEffectOnce(() => {
```

Comment:
why do we need useEffectOnce here?

Reply:
It is in a different pr for the Update sidebar